chaco/multi_line_plot.py

```diff
@@ -443,28 +439,23 @@ def _render_icon(self, gc, x, y, width, height):
     def _alpha_changed(self):
         self.invalidate_draw()
         self.request_redraw()
-        return
 
     def _color_changed(self):
         self.invalidate_draw()
         self.request_redraw()
-        return
 
     def _line_style_changed(self):
         self.invalidate_draw()
         self.request_redraw()
-        return
 
     def _line_width_changed(self):
         self.invalidate_draw()
         self.request_redraw()
-        return
```

Comment:
unrelated to this PR but looks like these could all use the new `requires_redraw` metadata

Reply:
can you please create a general purpose issue to search and see where all we could use `requires_redraw` in chaco?

Reply:
I've opened #595